chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/controller/ChaosMonkeyController.java

```diff
@@ -0,0 +1,22 @@
+package de.codecentric.spring.boot.chaos.monkey.controller;
+
+import de.codecentric.spring.boot.chaos.monkey.configuration.ChaosMonkeyProperties;
+import de.codecentric.spring.boot.chaos.monkey.configuration.ChaosMonkeySettings;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ChaosMonkeyController {
+
+    private final ChaosMonkeySettings chaosMonkeySettings;
+
+    public ChaosMonkeyController(ChaosMonkeySettings chaosMonkeySettings) {
+        this.chaosMonkeySettings = chaosMonkeySettings;
+    }
+
+    @PostMapping("/chaos-monkey/setting")
```

Comment:
@MrBW Not really sure about this name here... What do you think?

Reply:
Hmmm....what about:

-settings (it´s more than 1)
-configuration
-properties

I´m fine with all of them! ;-)